src/commands/transaction/accountlink.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ * Copyright 2019 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {AccountLinkTransaction, Deadline, TransactionHttp} from 'nem2-sdk';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'p',
+        description: 'The public key of the remote account',
+    })
+    publickey: string;
+    @option({
+        flag: 'a',
+        description: 'Alias action (0: Add, 1: Remove)',
+    })
+    action: number;
+
+    validateAction(value: number) {
```

Comment:
Think about using a validator. Example: https://github.com/nemtech/nem2-cli/blob/master/src/commands/profile/create.ts#L47

The validator should be tested.